.circleci/config.yml

```diff
@@ -69,4 +56,4 @@ workflows:
             - build
           filters:
             branches:
-              only: master
\ No newline at end of file
+              only: release
```

Comment:
Why release and not master?

Reply:
Good one, it should be master. Fixed it. was testing with the release branch of my fork